src/main/java/lotto/model/LottoNumberGenerator.java

```diff
@@ -5,20 +5,18 @@
 import java.util.List;
 
 public class LottoNumberGenerator {
-    static final String INPUT_MONEY_ERR_MSG = "구입 금액으로 숫자만 입력해야 합니다.";
-
     private final Payment payment;
 
     public LottoNumberGenerator(Payment payment) {
         this.payment = payment;
     }
 
     public Collection<LottoNumbers> generateLottoNumbersCollection() {
```

Comment:
```suggestion
    public Collection<LottoNumbers> generate() {
```

클래스명에도 이미 로또 번호라는 이름이 있으니 어떤 메서드인지만 잘 표현해도 충분하겠네요
메서드 이름은 간결한게 좋습니다.
다른 사람이 작성한 코드를 읽을 때 긴 단어, 문장보다 짤막하지만 뜻이 분명한 문장, 단어가 잘 읽히는 것을 비교해보아요